packages/form-footer/src/FormFooter.spec.tsx

```diff
@@ -3,10 +3,17 @@ import { render } from '@testing-library/react';
 import { axe } from 'jest-axe';
 
 import Button from '@leafygreen-ui/button';
+import X from '@leafygreen-ui/icon/dist/X';
 
 import { FormFooterProps } from './FormFooter';
 import FormFooter from '.';
 
+const buttonTestId = {
+  back: 'lg-form_footer-back-button',
```

Comment:
I believe these should all end with `_button` @TheSonOfThomp ?

Reply:
ooh gotcha. I was keeping it consistent with what was already in the file, but that makes sense. I'll keep an eye out for inconsistency with what shows up in the style guide and make updates when necessary